documentation/tutorials/getting-started-with-authentication.md

```diff
@@ -53,7 +53,7 @@ used when converting tokens or session information into a resource record.
 This authentication strategy provides registration and sign-in for users using a local
 identifier (eg `username`, `email` or `phone_number`) and a password.  It will
 define register and sign-in actions on your "user" resource.  You are welcome to
-define either or both of these actions yourself if you wish to customise them -
```

Comment:
This isn't a typo my guy 🤣

Reply:
Haha I just realized I've been using the British 🇬🇧 spelling.  Updated in 99bbcc4

Reply:
🙌